src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -1573,6 +1573,7 @@
   "Time": "Time",
   "TimeFormat": "Time Format",
   "TimeLeft": "Time Left",
+  "AddedOn": "Added On",
```

Comment:
What should I do for other languages? Plug it though Google Translate and say good enough?

Reply:
Don't do anything with other languages. The Key will be added to weblate and translators can add their translations.
please do sort your en.json though.

Reply:
Updated using VS's built in "sort keys" option -- there were a lot of changes . . .

Reply:
I think VS sorts badly compared to JetBrains Rider, as in it sorts capitals oddly. As you're only adding one Key it'd be easier just to add it in the right line :)

Reply:
![image](https://github.com/Sonarr/Sonarr/assets/14086872/62e0b8a2-c0e0-4864-a641-7d39e66a3bce)
That explains it -- yeah,VS was doing case insensitive.

Working on fixing tests too, will push up soon.

Reply:
Note that I left this as "AddedOn" rather than "Added" -- that seems right for the human-consumption string.